backend/fms_core/tests/test_template_importers/sample_submission_test.py

```diff
@@ -50,6 +50,7 @@ def test_import(self):
             sample = Sample.objects.get(name=sample_name)
             derived_sample_id = DerivedBySample.objects.filter(sample_id=sample.id).first().derived_sample_id
```

Comment:
My linter is complaining that `derived_sample_id` is not a property of DerivedBySample. Maybe this should be `first().derived_sample.id`? I don't know how the test could be passing though, if this is a real error...

Reply:
I traced the test in the debugger and derived_sample_id does seem to be returning id values, so the test is okay.

Reply:
This is one of Django magic trick. Each FK is in fact "attribute_name"_id in the DB and can be referred as such in the code. The advantage over derived_sample.id is that you do not need to join the derived_sample table to get the id, you have it on the derived_by_sample table.